plugins/woocommerce/src/Admin/WCAdminHelper.php

```diff
@@ -238,7 +238,7 @@ private static function get_normalized_url_path( $url ) {
 		$query           = wp_parse_url( $url, PHP_URL_QUERY );
 		$path            = wp_parse_url( $url, PHP_URL_PATH ) . ( $query ? '?' . $query : '' );
 		$home_path       = wp_parse_url( site_url(), PHP_URL_PATH );
-		$normalized_path = trim( substr( $path, strlen( $home_path ) ), '/' );
+		$normalized_path = trim( substr( $path, strlen( $home_path ?? '' ) ), '/' );
```

Comment:
This was giving error `Deprecated: strlen(): Passing null to parameter #1 ($string) of type string is deprecated in /var/www/html/wp-content/plugins/woocommerce/src/Admin/WCAdminHelper.php on line 241
`